packages/react/package.json

```diff
@@ -69,6 +69,42 @@
     "./dist/sass/References": {
       "sass": "./dist/sass/_References.scss"
     },
+    "./dist/sass/common": {
+      "sass": "./dist/sass/_common.scss"
+    },
+    "./dist/sass/constants": {
+      "sass": "./dist/sass/_constants.scss"
+    },
```

Comment:
I see the underscore in References, I think the only one that doesn't have it is Theming. IMO I would keep them, but idk if there's a reason why to remove them?

Reply:
after talking with David, he said he's comfortable here simply using a star export to make the entire sass folder accessible. That also means keeping the file names as is, without removing any underscores.